<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eccrypt</class>
 <widget class="QMainWindow" name="eccrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>eccrypt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/images.qrc">
    <normaloff>:/images/encryption.ico</normaloff>:/images/encryption.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>430</width>
     <height>310</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>430</width>
     <height>310</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_decryption">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>270</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Şifre Çöz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>235</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Ekle</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_languages">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>235</y>
      <width>185</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>410</width>
      <height>220</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_random">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>235</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rastgele</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>135</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçili Dosyaları Kaldır</string>
    </property>
    <property name="shortcut">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_encryption">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Şifrele</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
